daemon-tests/tests/rollover.rs

```diff
@@ -515,4 +482,22 @@ async fn rollover(
             .settlement_event_id,
         "Taker's latest event-id does not match given event-id"
     );
+
+    assert_ne!(
+        commit_tx_id_before_rollover_maker,
+        maker.latest_commit_txid(),
+        "The commit_txid of the taker after the rollover retry should have changed"
+    );
+
+    assert_ne!(
+        commit_tx_id_before_rollover_taker,
+        taker.latest_commit_txid(),
+        "The commit_txid of the maker after the rollover retry should have changed"
+    );
```

Comment:
👍 I'm happy that these were moved here.

🔧 But you have to remove the `retry` terminology, because this function is called in other contexts too.